test/jobs/github-event/pull_request/closed.js

```diff
@@ -60,12 +79,51 @@ test('github-event pull_request closed', async t => {
     t.ok(repository.enabled, 'repository is enabled')
     t.end()
   })
+
+  t.test('initial pr merged on private repo with payment plan', async t => {
```

Comment:
❓ why did you add this test?

Reply:
It wasn't tested before and since the special enterprise handling affects private repos, I wanted to have this as a test first, so I can check that the behaviour for private repos on enterprise is different.

Reply:
Thanks!